tasks.sh

```
@@ -100,16 +94,14 @@ function cmakeBuildAll() {
-        if ! cmd "/C cmake.exe --build . $TARGET_RULE $WIN_CMAKE_BUILD_DEFINE -- /m" && $EXIT_WHEN_FAILED; then
-            exit 1
-        fi
-        echo end: cmake.exe --build . $TARGET_RULE $WIN_CMAKE_BUILD_DEFINE -- /m
+        cmake.exe --build . $TARGET_RULE $WIN_CMAKE_BUILD_DEFINE -- /m
+        set +x
     else
-        if ! cmake --build . $TARGET_RULE $(test -n "$CMAKE_BUILD_TYPE" && echo --config $CMAKE_BUILD_TYPE) -- -j $(nproc) && $EXIT_WHEN_FAILED; then
-            exit 1
-        fi
+        set -x
+        cmake --build . $TARGET_RULE $(test -n "$CMAKE_BUILD_TYPE" && echo --config $CMAKE_BUILD_TYPE) -- -j $(nproc)
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)